src/components/inspectors/InspectorPanel.vue

```diff
@@ -76,7 +76,7 @@ export default {
   },
   computed: {
     highlightedNode() {
-      return store.getters.highlightedNode;
+      return store.getters.highlightedNodes[0];
```

Comment:
I like this - but once it's out, we can gather feedback on whether we actually want a default inspector panel when multiple shapes are selected.

Reply:
I did this mostly because it was the easiest way to prevent errors with the inspector with multiple shapes. Getting real user feedback on things like this would be very nice.

Another option would be to show some sort of common inspector or collection of things that can be applied to all selected shapes here.

Reply:
Yeah things we can discuss with Alan :)